storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/InsertQuery.java

```diff
@@ -7,7 +7,7 @@
 
 /**
  * Insert query for {@link com.pushtorefresh.storio.contentresolver.StorIOContentResolver}
- * <p>
+ * <p/>
```

Comment:
Is it Idea bug? Are there any differences with these tags. 
I see that `<p>` doesn't work when there isn't tag section in javadoc.
But works "better" in another cases =(